### PR TITLE
Test against Ruby 3.0.0,  Rails 6.1.0

### PR DESCRIPTION
**Pull request by tknzk**

### CODE REVIEW

File: saml_idp.gemspec
Comment (Zogoo) [inline saml_idp.gemspec:49]: Do we need to add "rexml" for Ruby 3.0. Because, I have still could see "rexml" in ruby 3.0 doc. Please correct me if I missing anything.
https://docs.ruby-lang.org/ja/latest/library/rexml=2fdocument.html

**Comment by Zogoo** [comment]
@jphenow I think we need to merge #144 before merge this pull request which makes correct order, because with this pull request Rails 5.1 test will be dropped.

File: saml_idp.gemspec
Comment (tknzk) [reply saml_idp.gemspec:49]: thanks.

please check this pull request. 
https://github.com/saml-idp/saml_idp/pull/155

rexml are not default gem (change to bundled gem) on ruby 2.8 (3.0) 
https://bugs.ruby-lang.org/issues/16485

**Review by jphenow — APPROVED** [review]

**Comment by jphenow** [comment]
Just waiting on the build then I'll merge and cut a release

File: gemfiles/rails_5.2.gemfile
Comment (mjobin-mdsol) [inline gemfiles/rails_5.2.gemfile:5]: this has no changes on CI, because `Gemfile.lock` isn't commited. 
it always pull the last version

it does not hurt though, just an extra commit for no reason

File: .travis.yml
Comment (mjobin-mdsol) [inline .travis.yml:20]: What? Why 5.2 was added to allowed failures?

**Pull request merged**
